velox/type/Timestamp.cpp

```diff
@@ -159,6 +159,17 @@ const date::time_zone& Timestamp::defaultTimezone() {
   return *kDefault;
 }
 
+int64_t Timestamp::getTimezoneOffset(
+    const Timestamp& timestamp,
+    const date::time_zone* timezone) {
+  auto tp = timestamp.toTimePoint();
+  auto localTime = date::make_zoned(timezone, tp);
+  auto utcTime = date::make_zoned(date::locate_zone("UTC"), tp);
```

Comment:
No need to make zoned for UTC, just use `tp` directly

Reply:
I changed it to `auto utcTime = date::make_zoned(tp);`